test/replicache_test.dart

```diff
@@ -38,14 +78,38 @@ Future<void> main() async {
 
   // TODO(arv): Start the test server from here!
 
+  List replays = [];
+  String recordPath;
+
+  Future<void> useReplay(String path) async {
```

Comment:
[help] Why have the replay mechanism? I thought that part of the goal was to test the entire stack, including replicache-client, so I'm surprised to see this here.

Reply:
I added it so that we can run the tests without working on the live data. Will be useful for CI. I'm fine removing this for now if we do not think it is worth keeping?